controllers/manila_controller.go

```diff
@@ -521,6 +530,52 @@ func (r *ManilaReconciler) reconcileNormal(ctx context.Context, instance *manila
 
 	// end transportURL
 
+	//
+	// create NotificationBus transportURL CR and get the actual URL from the
+	// associated secret that is created
+	//
+
+	// Request TransportURL when the parameter is provided in the CR
+	// and it does not match with the existing RabbitMqClusterName
+	if instance.Spec.NotificationBusInstance != nil {
+		// init .Status.NotificationURLSecret
+		instance.Status.NotificationURLSecret = ptr.To("")
```

Comment:
I was thinking we even have to init it here and not just do the assignment later when we have the `notificationBusInstanceURL.Status.SecretName`, but its fine to do it